tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_fillbug.py

```diff
@@ -0,0 +1,26 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+from gaiatest.apps.bzlite.app import BugzillaLite
+
```

Comment:
Pep8 requires 2 lines before class name after imports. You have one. Add one more line before line 5.